src/components/Button/style.js

```diff
@@ -220,16 +221,25 @@ export const large = css`
 `;
 
 const Sizes = {
-  small, medium, large
-}
+  small,
+  medium,
+  large,
+};
 
 /* state variants */
-export const disabled = css`
+export const secondaryDisabled = css`
```

Comment:
Just wondering if it might break any usages of a disabled button with a type that is not `primary` or `secondary`. A quick check through Publish, I see one usage of a `text` type button that could be disabled: https://github.com/bufferapp/buffer-publish/blob/56c1851a72c1a38da313c549dba8b2dbb3c46298/packages/post/components/PostFooterButtons/index.tsx#L45-L48

Reply:
Thank you @hyoyou for the comment 🌮 

I did another walk of the different combinations with Sofia and feels good to go now after some changes.